galleon-feature-pack/src/main/resources/modules/system/layers/base/org/jboss/resteasy/resteasy-spring/main/module.xml

```diff
@@ -28,5 +28,12 @@
     </resources>
 
     <dependencies>
+        <module name="javax.ws.rs.api"/>
+        <module name="org.jboss.resteasy.resteasy-client-api"/>
+        <module name="org.jboss.resteasy.resteasy-client" services="import"/>
+<!--        <module name="org.springframework.spring-core"/>-->
+<!--        <module name="org.springframework.spring-webmvc"/>-->
+        <module name="javax.servlet.api"/>
```

Comment:
This is similar to the situation above, in `pom.xml` it's:

```xml
<dependency>
            <groupId>org.jboss.spec.javax.servlet</groupId>
            <artifactId>jboss-servlet-api_4.0_spec</artifactId>
        </dependency>
```

Reply:
Same answer as above :)